contents/docs/self-host/deploy/upgrade-notes.md

```diff
@@ -132,3 +132,25 @@ If you didn’t make any customization to those, there’s nothing you need to d
     - `clickhouse.persistentVolumeClaim` -> `clickhouse.persistence.existingClaim`
 
     If you are overriding any of those, please make the corresponding changes before upgrading. Depending on your settings and setup, during this upgrade the ClickHouse pod might get recreated.
+
+### Upgrading from 10.x.x
+
+11.0.0 removes some legacy Helm annotations that are not necessary anymore (see [#179](https://github.com/PostHog/charts-clickhouse/pull/179) for more info).
+
+Before running the Helm upgrade, please run the following script first (note: replace the `RELEASE_NAME` and `RELEASE_NAMESPACE` accordingly if you are using a custom release name/namespace):
+
+```
+#!/usr/bin/env sh
+
+RELEASE_NAME="posthog"
+RELEASE_NAMESPACE="posthog"
+
+for deployment in $(kubectl -n "$RELEASE_NAMESPACE" get deployments --no-headers -o custom-columns=NAME:.metadata.name | grep "posthog")
```

Comment:
🤔 say e.g. I've added a metabase deployment or whatever to the posthog namespace, and called it posthog-metabase, is this something we want to consider?

Maybe we should explicitly name the resources just to be on the safe side, as isn't difficult to do?

Reply:
Fair, the case you described is a non-standard/supported setup but 👍

If we want to consider all cases I'm anyway not sure how we can provide a list of deployments. In a standard setup are something like `posthog-events`, `posthog-web` but users can always override them

Reply:
Ah yeah, the override bits, I hadn't seen that.

Maybe we just target standard setup, as a rule of thumb?

Reply:
Yeah that was my take